Assets/Scripts/UI/Components/HotKeyItem.cs

```diff
@@ -8,7 +8,21 @@ public class HotKeyItem : MonoBehaviour
     public Text itemText;
     public Button itemButton;
     public Text buttonText;
-    
+    public TooltipItem tooltip;
```

Comment:
add Init to this file

Reply:
that HotKeyManager handles the loading for us. or maybe i don't understand how it would work for this